experimenter/experimenter/nimbus-ui/package.json

```
@@ -54,7 +54,7 @@
     "react-select": "^5.7.3",
     "react-tooltip": "^4.5.0",
     "typescript": "4.3.4",
-    "vega": "^5.25.0",
+    "vega": "^5.22.0",
```

Can you add a second commit that pins vega to 5.22.0

```suggestion
    "vega": "5.22.0",
```